### ISSUE TITLE
Can I run tests using installed package

### ISSUE THREAD

**Issue by esc** [lmcinnes/umap#218]
After installing the package from conda-forge, is it possible to run the test-suite to see if the package was installed correctly?

**Comment by lmcinnes** [comment]
I don't believe the test suite is included in the conda-forge package. You should be able to try running any of the examples in the [documentation](https://umap-learn.readthedocs.io/en/latest/) as a simple test however.

**Comment by esc** [comment]
The reason I was asking about this, is because I have been working on an integration-testing suite to check umap against the latest Numba master. This acts as an early warning system for the Numba developers and will allow us to see if we are breaking anything:

https://github.com/numba/numba-integration-testing

If you do add support for running the tests from an installed package do let me know and I'll update that test-suite.

**Comment by lmcinnes** [comment]
Ah -- very useful to know. That certainly would motivate me to include the test suite in the installed package, although it does add dependencies on nose and a few other things that are possibly not ideal.

**Comment by sauln** [comment]
@esc, could you pull umap from the [github releases](https://github.com/lmcinnes/umap/releases) rather than conda-forge releases?  The github ones contain all the test code and should match.

**Comment by esc** [comment]
@sauln this is almost what I do, I actually pull the latest tag from the github repo. Details on what I do and how I compute the latest tag can be found here:

https://github.com/numba/numba-integration-testing/blob/master/switchboard.py#L178
